modules/auth/proc-enabling-authentication-with-rhsso.adoc

```diff
@@ -0,0 +1,72 @@
+[id="enabling-authentication-with-rhsso"]
+= Enabling authentication with Red Hat Single-Sign On (RHSSO)
+
+To authenticate users with Red Hat SSO (RHSSO), enable the OpenID Connect (OIDC) authentication provider in {product}.
+
+You might use this authentication provider to authenticate with other third-party services that support the OIDC protocol.
+
+.Prerequisites
+* You link:https://docs.redhat.com/en/documentation/red_hat_developer_hub/{product-version}/html/administration_guide_for_red_hat_developer_hub/assembly-add-custom-app-file-openshift_admin-rhdh[added a custom {product-short} application configuration], and have sufficient permissions to modify it.
+* You have sufficient permissions in RHSSO to create and manage a realm.
+
+.Procedure
+. To allow Developer Hub to authenticate with RHSSO, complete the steps in RHSSO, to link:https://docs.redhat.com/en/documentation/red_hat_single_sign-on/7.6/html-single/getting_started_guide/index#realms-apps_[create a realm and a user] and link:https://docs.redhat.com/en/documentation/red_hat_single_sign-on/7.6/html-single/getting_started_guide/index#registering-app_[register the {product-short} application]:
+.. Use an existing realm, or link:https://docs.redhat.com/en/documentation/red_hat_single_sign-on/7.6/html-single/getting_started_guide/index#create-realm_[create a realm], with a distinctive **Name** such as __<{product-very-short}>__.
+.. To register your {product-short} in RHSSO, in this realm, link:https://docs.redhat.com/en/documentation/red_hat_single_sign-on/7.6/html-single/getting_started_guide/index#registering-app_[create a Client ID], with:
+... **Client ID**: A distinctive client ID, such as __<{product-very-short}>__.
+//... **Client authentication**: Toggle to On.
+... **Valid redirect URIs**: Set to the OIDC handler URL: `https://__<RHDH_URL>__/api/auth/oidc/handler/frame`.
+... Navigate to the **Credentials** tab and copy the **Client secret**.
+... Save the **RHSSO URL**, **Realm name**, **Client ID**, and **Client Secret** values for the next step.
+
+.. To prepare for the verification steps, in the same realm, get the credential information for an existing user, or link:https://docs.redhat.com/en/documentation/red_hat_single_sign-on/7.6/html-single/getting_started_guide/index#create-user_[create a user], with:
+... **Username**: Set a distinctive user name, such as __<rhdhuser>__.
+... **Email verified**: Toggle to `On`.
+... To set a password for this user, navigate to the **Credentials** tab, click **Set password**, enter the **Password**, and toggle **Temporary** to `Off`.
+... Save the **Username** and **Password** for the verification steps.
+
+. To add your RHSSO credentials in your Developer Hub secrets, edit your Developer Hub secrets, such as `secrets-rhdh`, and add the following key/value pairs:
++
+`AUTH_OIDC_CLIENT_ID`:: Enter the Client ID that you generated in Keycloak, such as `RHDH`.
```

Comment:
Should we replace all occurrences of Keycloak with RHSSO?

Reply:
Absolutely!